README.md

```diff
@@ -144,13 +144,9 @@ Follow these steps only if your `src` folder is empty or you need the latest cod
 1. Commit the changes, and push extensions to GitHub.
 
 ### Update Dependencies
-1. (optional) Pull in the latest dependencies, including nested dependencies.
+1. Update the pinned requirements in `requirements-freeze.txt`. **Because of a version conflict for repoze.who, special care should be taken to make sure that repoze.who==1.0.18 is shipped to production in order to be compatible with ckanext-saml2. After generating the requirements-freeze.txt, manually review the file to make sure the versions are correct. See https://github.com/GSA/catalog-app/issues/78 for more details.** An initialized container will have the repoze.who version overwritten by the workaround script in entrypoint script, so we add `clean`, `build` to make sure the container is fresh. Then the dependencies are updated via `update-dependencies` and then added to the `requirements-freeze.txt` file:
 
-    $ make update-dependencies
-
-1. Update the pinned requirements in `requirements-freeze.txt`. **Because of a version conflict for repoze.who, special care should be taken to make sure that repoze.who==1.0.18 is shipped to production in order to be compatible with ckanext-saml2. After generating the requirements-freeze.txt, manually review the file to make sure the versions are correct. See https://github.com/GSA/catalog-app/issues/78 for more details.** An initialized container will have the repoze.who version overwritten by the workaround script in entrypoint script, so we add `clean`, `build` to make sure the container is fresh. 
-
-    $ make clean build requirements
+    $ make clean build update-dependencies requirements
```

Comment:
This is more correct, but I think the `update-dependencies` task isn't necessary. requirements.txt gets installed at container build time (`make build`) from the Dockerfile. At that point running `pip install -r requirements.txt` (`make update-dependencies`) is redundant?

But then why even have the update-dependencies at all? Maybe if you already have your container running and you `git pull` that brings in an updated requirements.txt, thats when you'd want to run `make update-dependencies`? But then I feel like maybe you should just rebuild the containers `make clean build` at that point. Idk.

Reply:
Probably is redundant, could simplify. But I confirmed it works as we would like.

Reply:
🤔  Or.... maybe Dockerfile should use requirements-freeze.txt... Realizing now that if git dependencies have updates, requirements.txt doesn't change so docker won't rebuild the container (it think's there's nothing to update and will use the cached images). But by making it install from requirements-freeze, any change would trigger the docker build as expected.

In that world, update-dependencies makes sense because you're saying, rebuild all my dependencies from spec (requirements.txt) so they can then be frozen.